kafka/src/main/java/io/micronaut/configuration/kafka/processor/KafkaConsumerProcessor.java

```diff
@@ -97,19 +97,7 @@
 import javax.annotation.PreDestroy;
 import java.nio.charset.StandardCharsets;
 import java.time.Duration;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Locale;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Properties;
-import java.util.Set;
-import java.util.UUID;
+import java.util.*;
```

Comment:
Could you revert this an not use star imports?

Reply:
Sure

Reply:
But it's better to add this rule to the style checking in the future :)